packages/next/client/index.js

```diff
@@ -371,8 +371,8 @@ function renderReactElement(reactEl, domEl) {
       const opts = { hydrate: true }
       reactRoot =
         process.env.__NEXT_REACT_MODE === 'concurrent'
-          ? ReactDOM.createRoot(domEl, opts)
-          : ReactDOM.createBlockingRoot(domEl, opts)
+          ? ReactDOM.unstable_createRoot(domEl, opts)
```

Comment:
Won't this break on versions before `33c3af284`? I'm not sure it is worth it, but what about:

```
const createRoot = ReactDOM.unstable_createRoot ? ReactDOM.unstable_createRoot : ReactDOM.createRoot
cost createBlockingRoot = ReactDOM.unstable_createBlockingRoot ? ReactDOM.unstable_createBlockingRoot : ReactDOM.createBlockingRoot
```

Reply:
> Won't this break on versions before 33c3af284? 

Yes, which is IMO fine for experimental features. I think maintaining multiple experimental versions is not viable long-term. But that's up to the maintainers.

Reply:
👍 just thought I'd ask =)

Reply:
Yep we should just update it 👍